src/test/expression/lqp_select_expression_test.cpp

```diff
@@ -18,7 +18,7 @@ using namespace opossum::expression_functional;  // NOLINT
 
 namespace opossum {
 
-class LQPSelectExpressionTest : public ::testing::Test {
```

Comment:
On a more serious note: Should we add `::testing::Test` to the linter or do we hope that now that we got rid of all remains, no one will reintroduce it?

Reply:
I think there are still some simple tests (not loading any tables) that still derive from ::testing::Test. Up to you

Reply:
ok